### PR TITLE
Prevent Unbuckle and ToggleInternal actions while Incapacitated

### PR DESCRIPTION
**Pull request by tomasalves8**
## About the PR 

This prevents players from unbuckling or toggling internals while dead or incapacitated.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: Players can no longer unbuckle or toggle internals while dead or in critical condition

### CODE REVIEW

**Comment by Elijahrane** [comment]
Consider checking actionblocker instead, which will take into account mobstate and other stuff that should block it, too

**Comment by tomasalves8** [comment]
I'll have a look now.

**Comment by tomasalves8** [comment]
Something like this?

**Comment by Elijahrane** [comment]
yeah that looks correct

File: Content.Server/Body/Systems/InternalsSystem.cs [hunk 1]
```diff
@@ -11,6 +11,9 @@
 using Content.Server.Popups;
 using Content.Server.DoAfter;
 using System.Threading;
+using Content.Shared.ActionBlocker;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs.Systems;
```
Comment (Elijahrane) [inline Content.Server/Body/Systems/InternalsSystem.cs]: the 2 mob usings here are unused.

if you have an IDE, unused using will usually be greyed out.

If you do not, I recommend finding some tool for C# for whatever you prefer. Personally I use VS Code although I know there are independent plugins for lighter stuff like vim

**Review by Elijahrane — CHANGES_REQUESTED** [review]
just remove those 2 usings and I'll merge

**Pull request merged**